samples/suit/smp_transfer/sysbuild/nrf54h20dk_nrf54h20_memory_map.dtsi

```diff
@@ -4,40 +4,15 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
  */
 
-/delete-node/ &cpuapp_rw_partitions;
-
-&mram1x {
-	erase-block-size = < 0x10 >;
-	/* Hardcoded inside the soc_flash_nrf_mram.c (MRAM_WORD_SIZE) */
-	write-block-size = < 0x10 >;
-
-	cpuapp_rw_partitions: cpuapp-rw-partitions {
-		compatible = "nordic,owned-partitions", "fixed-partitions";
-		status = "okay";
-		perm-read;
-		perm-write;
-		#address-cells = < 0x1 >;
-		#size-cells = < 0x1 >;
-
-		dfu_partition: partition@136000 {
-			reg = < 0x136000 DT_SIZE_K(656) >;
-		};
-
-		storage_partition: partition@1da000 {
-			reg = < 0x1da000 DT_SIZE_K(24) >;
-		};
-	};
-};
-
 &cpuapp_rx_partitions {
 	/delete-node/ partition@a6000;
 
 	cpuapp_slot0_partition: partition@a6000 {
-		reg = <0xa6000 DT_SIZE_K(448)>;
+		reg = <0xa6000 DT_SIZE_K(260)>;
```

Comment:
Please change the addresses and sizes to match what is added in https://github.com/nrfconnect/sdk-nrf/pull/15422 - otherwise we can potentially override the configuration by accident

Reply:
Aligned!